src/main/java/application/rest/v1/ComponentInfoRegistry.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2019 IBM Corporation
+ * Copyright 2020 IBM Corporation
```

Comment:
Copyright statements need first year and year with latest changes, so this should be "Copyright 2019, 2020 IBM Corporation".

Reply:
Updated.